src/index.d.ts

```diff
@@ -0,0 +1,21 @@
+type parseOptions = {
+  hiddenPaths?: string[];
+  matchesNode?: boolean;
+  maxPathLength?: number;
+  maxSourceLines?: number;
+  useColor?: boolean;
+  verbose?: boolean;
+}
+
+declare class UML {
+  constructor(elements: object[]);
```

Comment:
If the type declaration could further specify what `elements` is instead of just `object`, meaning a ts-type per element type [here](https://github.com/Enteee/plantuml-parser/tree/master/src), that would be Ideal.

Reply:
I've amended this but have needed to add `object[]` in other places, see latest commit.